aii-opennebula/src/main/resources/vmtemplate.tt

```diff
@@ -47,7 +47,7 @@ IMAGE_UNAME = "oneadmin"
 GRAPHICS = [
 [%     FILTER indent -%]
 LISTEN = "0.0.0.0",
-TYPE = "VNC"
+TYPE = "[% system.opennebula.graphics %]"
```

Comment:
there is no extra regexp lines to test this, eg with `SPICE` instead of old default